docs/development.md

```diff
@@ -67,93 +106,147 @@ docker compose down -v
-```
-
-In order to populate the database, you must create a `.env` file in the top
-level directory containing mongo credentials.
-
-```bash
-# .env
-NMDC_MONGO_USER=changeme
-NMDC_MONGO_PASSWORD=changeme
-```
-
-With that file in place, populate the docker volume by running,
-
-```bash
-docker-compose run backend nmdc-server truncate # if necessary
-docker-compose run backend nmdc-server migrate
-docker-compose run backend nmdc-server ingest -vv --function-limit 100
-```
-
-# Running the client
-
-Run the client in development mode.
-
-``` bash
-cd web/
-yarn
+cd web
 yarn serve
 ```
 
-View main application at `http://127.0.0.1:8081`
+View the main application at `http://127.0.0.1:8081/`. Changes to files in the `web` directory will automatically trigger a reload in your browser. 
+
+> **Note**: the frontend application will still be served via Docker Compose on port `8080`, but it will not pick up changes to the `web` directory automatically. Be aware of which port you are accessing when doing frontend development.
 
-## Why not `localhost`?
+### Why not `localhost`?
 
 It is recommended to use `127.0.0.1` instead of `localhost` for local development. This is because `localhost` is **not** allowed as a redirect URI for an ORCID client. The workaround is to register `127.0.0.1` as a redirect URI with ORCID and to use subsequently visit `127.0.0.1` for local testing.
 
-# Testing
+## Running ingest
+
+> **Note**: This is not generally required unless you are specifically working on the ingest code. If you are working on the web application, simply [loading from a recent production backup](#load-production-data) is sufficient.
+
+1. Ensure that you have completed the sections above of configuring [NERSC credentials](#nersc-credentials) and [MongoDB credentials](#mongodb-credentials). 
+2. Obtain a new SSH key from NERSC if you haven't done so in the last 24 hours.
+    ```bash
+    sshproxy.sh -u $NERSC_USER
+    ```
+3. Set up an active SSH tunnel to the dev and production MongoDB instances if you do not already have one established.
+    ```bash
+    ssh \
+      -L 37018:mongo-loadbalancer.nmdc-dev.production.svc.spin.nersc.org:27017 \
+      -L 37019:mongo-loadbalancer.nmdc.production.svc.spin.nersc.org:27017 \
+      -o ServerAliveInterval=60 \
+      -f \
+      -N \
+      -l $NERSC_USER \
+      -i ~/.ssh/nersc \
+      dtn01.nersc.gov
+    ```
+    > That command will set up SSH port forwarding such that your computer can access the dev MongoDB server at `localhost:37018` and the prod MongoDB server at `localhost:37019`. 
+  
+    > From within a Docker container `host.docker.internal` can be used to access the `localhost` of your computer. When ingesting from the dev or prod MongoDB instances, be sure to set `NMDC_MONGO_HOST=host.docker.internal` in your `.env` file.
+
+    > See https://github.com/microbiomedata/infra-admin/blob/main/mongodb/connection-guide.md for more information on connecting to the MongoDB instances.
```

Comment:
I added "(internal)" so that non-NMDC people don't think the referenced documentation has been deleted (it is in a private repo).

```suggestion
    > See https://github.com/microbiomedata/infra-admin/blob/main/mongodb/connection-guide.md (internal) for more information on connecting to the MongoDB instances.
```